plugin/src/Plugin.php

```diff
@@ -89,4 +90,15 @@ public function set_user( $args, array $assoc_args ): void {
 		file_put_contents( $user_id_file, $user_id );
 		WP_CLI::success( 'Current User set to ' . $args[0] );
 	}
+
+	public function initialize_admin_user_meta(): void {
+		global $wpdb;
+		$user_preferences_meta_key = $wpdb->get_blog_prefix() . 'persisted_preferences';
+
+		update_user_meta(1, $user_preferences_meta_key, [
+			"core/edit-post" => [
+				"welcomeGuide" => false,
+			],
+		]);
+	}
```

Comment:
this runs on each `init` which makes it impossible to update the meta to a different value. I'm not aware of a suitable hook to update meta on the root/admin user as they're handled differently to other users, so perhaps we should check for the presence of existing meta first and only update if none is found